test/factory/listing.go

```diff
@@ -109,32 +109,13 @@ func NewCryptoListing(slug string) *pb.Listing {
 	return listing
 }
 
-func NewShippingRegionListing(slug string, countrycode pb.CountryCode) *pb.Listing {
+func NewListingWithShippingRegions(slug string) *pb.Listing {
 	listing := NewListing(slug)
 	listing.ShippingOptions = []*pb.Listing_ShippingOption{
 		{
 			Name:    "usps",
 			Type:    pb.Listing_ShippingOption_FIXED_PRICE,
-			Regions: []pb.CountryCode{countrycode},
-			Services: []*pb.Listing_ShippingOption_Service{
-				{
-					Name:              "standard",
-					Price:             20,
-					EstimatedDelivery: "3 days",
-				},
-			},
-		},
-	}
-	return listing
-}
-
-func NewShippingRegionsProtoBufAlias(slug string, countrycodes []pb.CountryCode) *pb.Listing {
-	listing := NewListing(slug)
-	listing.ShippingOptions = []*pb.Listing_ShippingOption{
-		{
-			Name:    "usps",
-			Type:    pb.Listing_ShippingOption_FIXED_PRICE,
-			Regions: countrycodes,
```

Comment:
See the commit message for the rationale here. The two factories were quite similar and I was able to reduce these two functions to just one extra line after the factory's result in the unit test itself.